Formula/p/passt.rb

```diff
@@ -0,0 +1,31 @@
+class Passt < Formula
+  desc "User-mode networking daemons for virtual machines and namespaces"
+  homepage "https://passt.top/passt/about/"
+  url "https://passt.top/passt/snapshot/passt-2024_07_26.57a21d2.tar.xz"
+  version "2024_07_26.57a21d2"
+  sha256 "8237f5af399ebd49a6946954dc4cc2cbca49c69bfa33a0cf39ac5cfd04d9ddde"
+  license all_of: ["GPL-2.0-or-later", "BSD-3-Clause"]
+  head "git://passt.top/passt", branch: "master"
+
+  depends_on :linux
+
+  def install
+    args = ["prefix=#{prefix}"]
+    args << "VERSION=#{version}" if build.stable?
+    system "make", "install", *args
```

Comment:
These paths need to be set so that everything is installed to the formula dir

```suggestion
    args = ["prefix=#{prefix}",  "bindir=bin", "man1dir=share/man", "docdir=share/doc"]
    args << "VERSION=#{version}" if build.stable?
    system "make", "install", *args
```

Reply:
The variables should already handle this as `prefix` controls the main prefix and will overwrite default at https://passt.top/passt/tree/Makefile?h=2024_07_26.57a21d2#n92
```
prefix		?= /usr/local
exec_prefix	?= $(prefix)
bindir		?= $(exec_prefix)/bin
datarootdir	?= $(prefix)/share
docdir		?= $(datarootdir)/doc/passt
mandir		?= $(datarootdir)/man
man1dir		?= $(mandir)/man1
```

It works as expected when tested on my Linux container, e.g.
```console
$ tree ~/.linuxbrew/opt/passt
/home/linuxbrew/.linuxbrew/opt/passt
|-- INSTALL_RECEIPT.json
|-- README.md
|-- bin
|   |-- passt
|   |-- pasta -> passt
|   `-- qrap
|-- sbom.spdx.json
`-- share
    |-- doc
    |   `-- passt
    |       |-- README.md
    |       `-- demo.sh
    `-- man
        `-- man1
            |-- passt.1
            |-- pasta.1 -> passt.1
            `-- qrap.1
```